installer/templates/phx_web/templates/layout/live.html.heex

```diff
@@ -1,11 +1,19 @@
 <main class="container">
-  <p class="alert alert-info" role="alert"
+  <p
+    class="alert alert-info"
+    role="alert"
     phx-click="lv:clear-flash"
-    phx-value-key="info"><%%= live_flash(@flash, :info) %></p>
+    phx-value-key="info"
+    phx-no-format
```

Comment:
That's required while https://github.com/phoenixframework/phoenix/pull/4667 isn't released.